home/src/main/resources/rdf/i18n/de_DE/display/firsttime/search_individuals_vitro.n3

```diff
@@ -12,3 +12,7 @@
 :field_type rdfs:label "Typ"@de-DE .
 :filter_querytext rdfs:label "Text"@de-DE .
 :field_querytext rdfs:label "Standardfeld"@de-DE .
+:sort_name_raw_desc rdfs:label "Roher Titel Z-A"@de-DE .
+:sort_name_raw_asc rdfs:label "Roher Titel A-Z"@de-DE .
```

Comment:
vitro-search:display property for sort_name_raw_desc is missing, meaning it is false by default, meaning with the current configuration these labels are not needed. I think if someone decide to turn it on (set display to true), he/she should add labels for languages he/she needs. Not sure we need this in the codebase? The comment is the same for all other languages for those two labels.

Reply:
In first version of extended search configuration was stored in content store, so users could navigate through configuration. Labels will be useful when we create an option to navigate over search settings in display graph.